base/complex.jl

```diff
@@ -535,37 +535,39 @@ end
 #     return Complex(abs(iz)/r/2, copysign(r,iz))
 # end
 
-# compute exp(im*theta)
-function cis(theta::Real)
-    s, c = sincos(theta)
-    Complex(c, s)
-end
-
 """
-    cis(z)
+    cis(x)
 
-Return ``\\exp(iz)``.
+Similar to `exp(im*x)`, this computes ``\\exp(i x)``, but is typically faster.
```

Comment:
This still seems a bit confusing, why write the same formula twice in a sentence, and what exactly does similar mean, does it mean equal? 

My earlier suggestion was to separate two things -- the first line tells you what it returns, that's the basic mathematical  fact. The next paragraph tries to explain why this exists instead of `exp(im*z)`, and hence why you may want to use it, which is software detail.

Reply:
The reason for stating it twice is once in Julia syntax and once in math/latex/textbook syntax so a reader will understand regardless of what background they are coming from. Also Latex doesn't render in the REPL so having the Julia syntax is helpful.

I agree that "Similar" is a bit confusing as it implies it is not the same.  I think it is the same up to 0.5 ULP so I changed it to "Same as"